cluster/gce/util.sh

```diff
@@ -2046,29 +2047,29 @@ function create-node-template() {
 
   local gcloud="gcloud"
 
-  local accelerator_args=""
+  local accelerator_args=()
   # VMs with Accelerators cannot be live migrated.
   # More details here - https://cloud.google.com/compute/docs/gpus/add-gpus#create-new-gpu-instance
-  if [[ ! -z "${NODE_ACCELERATORS}" ]]; then
-    accelerator_args="--maintenance-policy TERMINATE --restart-on-failure --accelerator ${NODE_ACCELERATORS}"
+  if [[ -n "${NODE_ACCELERATORS}" ]]; then
+    accelerator_args+=(--maintenance-policy TERMINATE --restart-on-failure --accelerator "${NODE_ACCELERATORS}")
```

Comment:
Should `kube::util::read-array` be used here for uniformity, or is it purposefully not used here?

Reply:
`kube::util::read-array` reads line by line and adds it to the array, while here it is a single line that is added. I consider these two separate use cases where `kube::util::read-array` is the best solution for the case for line-by-line, being Bash 3 backwards compatible and all, while this way of writing is the easiest to use/follow/maintain for the single line case.